Scripts/Services/Revamped Dungeons/WrongDungeon/Mobile/GooeyMaggots.cs

```diff
@@ -37,13 +38,47 @@ public GooeyMaggots()
             this.Karma = -1000;
 
             this.VirtualArmor = 24;
+
+            Timer SelfDeleteTimer = new InternalSelfDeleteTimer(this);
```

Comment:
Prefer camel case for local variables: `selfDeleteTimet` over `SelfDeleteTimer`.